src/backend/commands/matview.c

```diff
@@ -440,6 +440,17 @@ refresh_matview_datafill(DestReceiver *dest, Query *query,
 								GetActiveSnapshot(), InvalidSnapshot,
 								dest, NULL, 0);
 
+	if (gp_enable_gpperfmon && Gp_role == GP_ROLE_DISPATCH)
+	{
+		Assert(queryString);
+		gpmon_qlog_query_submit(queryDesc->gpmon_pkt);
+		gpmon_qlog_query_text(queryDesc->gpmon_pkt,
+				queryString,
+				application_name,
+				NULL,
```

Comment:
Why is there no information about resource queues like above?

Reply:
It's about QueryDesc. As I remember, you won't be able to use `gpmon_qlog_packet_init` with QueryDesc created with CreateUtilityQueryDesc

Reply:
What? The query desc is created by the regular CreateQueryDesc here.

Reply:
Sorry, misunderstood your question. I added resqueue logging for matview